source_modelling/sources.py

```diff
@@ -76,56 +76,27 @@ def from_lat_lon_depth(point_coordinates: np.ndarray, **kwargs) -> "Point":
 
     @property
     def coordinates(self) -> np.ndarray:
-        """Return the coordinates of the point in (lat, lon, depth) format.
-
-        Returns
-        -------
-        np.ndarray
-            The coordinates of the point in (lat, lon, depth) format.
-            Depth is in metres.
-        """
+        """np.ndarray: The coordinates of the point in (lat, lon, depth) format. Depth is in metres."""
```

Comment:
Is this a recommended docstring format in case of no parameters?

Reply:
Yes, this is actually the numpydoc standard format for all decorated `@property` functions. See https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html.